examples/servo/build.rs

```diff
@@ -18,16 +16,21 @@ fn main() {
     let out = env::var("OUT_DIR").unwrap();
     let out = Path::new(&out);
 
-    #[cfg(target_os = "linux")]
+    let target_os = std::env::var("CARGO_CFG_TARGET_OS").unwrap();
+
     {
         let mut file = File::create(&out.join("gl_bindings.rs")).unwrap();
 
         // Config copied from https://github.com/YaLTeR/bxt-rs/blob/9f621251b8ce5c2af00b67d2feab731e48d1dae9/build.rs.
 
+        let api = if target_os == "android" { Api::Gles2 } else { Api::Gl };
```

Comment:
In this code, the decision is "sort of" made at run-time by comparing strings. However, this can be done at compile-time, using either `#[cfg(target_os = "android"]` or using `if cfg!(target_os = "android")`.

Given that either branch is compilable, I think this should be written like this:

```rust
let api = if cfg!(target_os = "android") { ... } else { ... };
```

That said, I wonder if we can get away with just OpenGL ES 3.0 everywhere. On Android and Linux Embedded that'll be preferred. On Windows I thought servo is using angle, so that's also OpenGL ES.

@expenses I'm unsure, but is there perhaps a reason to use "Desktop GL" over GLES here?